windows/Dockerfile

```diff
@@ -30,6 +30,14 @@ RUN `
     && (for %i in ("%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\*") do if not "%~nxi" == "vswhere.exe" del "%~i") `
     && powershell Remove-Item -Force -Recurse "%TEMP%\*"
 
+# Install supported .NET SDKs
+RUN `
+    curl -fSLO https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1 `
+    && ./dotnet-install.ps1 -Channel 6.0 `
```

Comment:
`.\dotnet-install.ps1` for Windows?

Reply:
The current shell is `cmd` so you might also need to explicitly use `powershell` to run this?

Reply:
Could move line 43 above this to switch shells.

Reply:
Then use `;` to chain commands if you do that.